src/EditorFeatures/CSharpTest2/Recommendations/SByteKeywordRecommenderTests.cs

```diff
@@ -623,5 +623,30 @@ static void Main(string[] args)
 }
 ", matchPriority: SymbolMatchPriority.Keyword);
         }
+
+        [WorkItem(14127, "https://github.com/dotnet/roslyn/issues/14127")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestInTupleWithinType()
+        {
+            await VerifyKeywordAsync(@"
+class Program
+{
+    ($$
+}");
+        }
+
+        [WorkItem(14127, "https://github.com/dotnet/roslyn/issues/14127")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestInTupleWithinMember()
+        {
+            await VerifyKeywordAsync(@"
+class Program
+{
+    void Method()
+    {
+        ($$
+    }
+}");
```

Comment:
Also need tests for compound foreach.  Also, at least a test or so for tuples in other areas (parameter lists, etc.)